nextjs-13/src/components/PublicLayout/index.tsx

```diff
@@ -0,0 +1,24 @@
+'use client';
+
+import Link from 'next/link';
+import { useSession } from 'next-auth/react';
+
+export default function PublicLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const { status } = useSession();
+
+  return (
+    <div className="public" data-testid="public-newsletter">
```

Comment:
Avoid single-word class names to avoid class conflicts

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/127/commits)